packages/malloy-db-bigquery/src/bigquery.spec.ts

```diff
@@ -28,6 +28,17 @@ import * as util from 'util';
 import * as fs from 'fs';
 import {fileURLToPath} from 'url';
 
+const envDatabases = (
```

Comment:
Since this gesture is so common maybe it could be

```Typescript
envDatabases = getEnvDatabases(default | undefined)
```

?

Reply:
There's a bunch of repeated code in our connections that might benefit from some sort of core utilities library. Packages aren't suppose to reach out and pull in code from other directories without a package dependency (although we may not care that much, I guess), but the only package they share is @malloydata/malloy, and I feel like adding test helpers there might just be bloat.